src/Asset/Capacity/HasAntivirusCapacity.php

```diff
@@ -51,6 +51,11 @@ public function getSearchOptions(string $classname): array
         return ItemAntivirus::rawSearchOptionsToAdd();
     }
 
+    public function getDescription(): string
```

Comment:
A general remark here, we merged last week the PHP 8.3 polyfill, which mean we should try to use the `#[Override]` data attribute every time we override a method.

See https://stitcher.io/blog/override-in-php-83